cleanlab/multiannotator.py

```diff
@@ -743,13 +792,23 @@ def get_active_learning_scores_ensemble(
                 "or just pass in pred_probs_unlabeled to get active learning scores for unlabeled examples.",
             )
 
-        if isinstance(labels_multiannotator, np.ndarray):
-            labels_multiannotator = pd.DataFrame(labels_multiannotator)
+        if isinstance(labels_multiannotator, pd.DataFrame):
+            labels_multiannotator = (
+                labels_multiannotator.replace({pd.NA: np.NaN}).astype(float).to_numpy()
+            )
+        elif isinstance(labels_multiannotator, np.ndarray):
+            # make sure it is a column array
+            if labels_multiannotator.ndim == 1:
+                labels_multiannotator = labels_multiannotator.reshape(-1, 1)
```

Comment:
should this line be unit-tested?

Reply:
This is non-intended, have edited this line